src/Analytics/Adapter.php

```diff
@@ -124,7 +124,7 @@ public function call(string $method, string $path = '', array $headers = [], arr
         $responseType = '';
         $responseBody = '';
 
-        switch ($headers['Content-Type']) {
+        switch ($headers['Content-Type'] ?? '') {
```

Comment:
Do the the responses not have a content-type? Can you check and see if it's there but lowercase instead?